web/client-ide/nginx/nginx.conf

```diff
@@ -0,0 +1,35 @@
+user  nginx;
+worker_processes  1;
+
+# This is important! because it loads several dav module with this.
+include /etc/nginx/modules-enabled/*.conf;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
```

Comment:
Does nginx actually write out to this location when running in docker, or does everything go to stdout?

Eventually, we'll want to audit where the containers are writing to - https://github.com/docker-library/docs/tree/master/nginx#running-nginx-in-read-only-mode my be relevant.

Reply:
It looks like it's writing to those paths, but those paths are mapped to stdout: 
```
root@2de6289d3cd6:/# ls -la /var/log/nginx/
total 8
drwxr-xr-x 1 root adm  4096 Feb 17 19:20 .
drwxr-xr-x 1 root root 4096 Feb 17 19:20 ..
lrwxrwxrwx 1 root root   11 Feb 17 19:20 access.log -> /dev/stdout
lrwxrwxrwx 1 root root   11 Feb 17 19:20 error.log -> /dev/stderr
```

It writes to the nginx.pid.

Should we have a ticket to audit where it writes? It seems like a different scope than this ticket